### ISSUE TITLE
Can we create a feedback/suggest a correction button for DPUL similar to that of FInding Aids site?

### ISSUE THREAD

**Issue by kelea99** [pulibrary/pomegranate#812]
@kelynch and I discussed this briefly as well.  

Met  with Ellen A (South Asian Ephemera) and Gabrielle W. (Latin American Ephemera) about ephemera collections. IT would be great to have a button similar to the Finding Aid site that would open a form for suggestions to enhance metadata or feedback to alert staff to mistakes or corrections needed to metadata and/or images.

A feedback mechanism of some sort (perhaps exactly this?) was also discussed with Emma S. of SC (Coordinator of the Curated Collections program).

**Comment by tpendragon** [comment]
Yeah, this is an old ticket: https://github.com/pulibrary/pomegranate/issues/285.

**Comment by escowles** [comment]
Here's a screenshot of the Finding Aids "Suggest a correction" form: 
![Screen Shot 2020-07-20 at 2 26 08 PM](https://user-images.githubusercontent.com/856924/87972514-03bedf80-ca95-11ea-8f6f-e6f85389d67a.png)

**Comment by kelea99** [comment]
I like that, @escowles . It's clean and we would want something relatively simple to try out.  Box/container number wouldn't be needed...actually. The space could be repurposed for DPUL, perhaps to a dropdown? "feedback", "suggest a correction", whatever -- so that it could be sent to the appropriate people? what do yo think?

**Comment by hackmastera** [comment]
This seems like a different issue from #285. That one is a general feedback form as in the header on the catalog; this one is specific to a given record. What email address should this one submit to @kelea99 ?

**Comment by kelea99** [comment]
@hackmastera it depends on the item and I would really like to hear your thoughts. I see  a couple of ways this can go--one is a bit of a pain, but more direct. the other seems like it would be programmatically easier (but I do not know)?
1. (Easier?): the correction suggestion goes to the exhibition owner(s).
2. The Correction Suggestion, depending on what the item/correction is, would be triaged:

**The metadata is incorrect in the DPUL/Figgy/Online Catalog record:**

- Any corrections requests for MARC/catalog records may be sent to: catalogn@princeton.edu

- Any corrections requests for EAD/Finding Aids: click on “suggest a correction” within the finding aid component itself or wherever that form is sent to ( would Kevin know this?)

- Any corrections requests for ephemeral items would go to the Selector who works with the ephemeral collection (examples include Gabrielle/Fernando for LAE, Thomas/Anna for Soviet, Ellen for SAE, etc.)

**If an item is described in an exhibition in DPUL incorrectly (in a curators note, or text within and exhibition, for example)**
- Contact Kim Leaman (kleaman@princeton.edu / Slack: Kelea) and/or the exhibition curator listed in the left hand column on the “About” pages within the exhibit.

**If the wrong metadata is associated with an item in Figgy and/or DPUL.** 
- Contact Kim Leaman (kleaman@princeton.edu / Slack: Kelea), and provide a link to the item in figgy.
**If the images are oriented incorrectly in the viewer, pages are askew and/or missing in Figgy.**
- Contact Kim Leaman (kleaman@princeton.edu / Slack: Kelea) or Roel Munoz (rmunoz@princeton.edu / Slack: Roel) and provide a link to the figgy/online catalog/DPUL URL, along with a description as to what is wrong with the images.

**Comment by hackmastera** [comment]
@kelea99 Either way works and it seems like a discussion to have with the exhibition owners. Option 1 we would do by pulling the email addresses from the exhibit record. Option 2 we would do by sending them all to a single / central email address (probably a metadata-oriented libguides email address?); and then the person monitoring that address would follow the workflow you lay out. We would just need to know what the email address is to send to.

**Comment by hackmastera** [comment]
Sorry I said libguides but I meant libanswers.
